1-basics/maxAll.js

```diff
@@ -5,7 +5,14 @@
  * @return {Number} The max of the numbers
  */
 function maxAll(numbers) {
-  // your code here
+  Number = numbers[0]
```

Comment:
1- in order to create a new variable, you need to use the `let` keyword like 
```js
let varName = value
```
2- In order to make the difference between variables and classes (we will learn what are classes in the future), we start the variables names with lower case characters:
```js
luffy // good variable name
Luffy // bad variable name
luffyisbaaka // bad name because difficult to read
luffyIsBaaka // good, this is called camel case (start each word with an upper case character except the first one)
luffy_is_baaka // good too, this is called snake case
```

So instead of doing
```js
Number = numbers[0]
```
you should do
```js
let number = numbers[0]
```
or even better
```js
let max = numbers[0] // this is better because it gives information about what the variable is used for
```